content/docs/engine-dev/tools/project-manager/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+linkTitle: Project Manager
+title: Project Manager Developer Documentation
+description: Developer documentation for the Project Manager tool.
+---
+
+{{< note >}}
+This information is for developers of the **Project Manager** tool. If you're a user working with Project Manager, please refer to the [Project Manager User Guide](/docs/user-guide/project-config/project-manager).
```

Comment:
nit - do we typically use conjunctions in O3DE documentation?  Unsure, just checking.

Reply:
Yes, it was encouraged by AWS editors.